{
    "containers": {
        "cna": {
            "affected": [
                {
                    "product": "Data Risk Manager",
                    "vendor": "IBM",
                    "versions": [
                        {
                            "status": "affected",
                            "version": "2.0.6"
                        }
                    ]
                }
            ],
            "datePublic": "2020-09-21T00:00:00",
            "descriptions": [
                {
                    "lang": "en",
                    "value": "IBM Data Risk Manager (iDNA) 2.0.6 could allow an authenticated user to obtain sensitive information using a specially crafted HTTP request. IBM X-Force ID: 184924."
                }
            ],
            "metrics": [
                {
                    "cvssV3_0": {
                        "attackComplexity": "LOW",
                        "attackVector": "NETWORK",
                        "availabilityImpact": "NONE",
                        "baseScore": 4.3,
                        "baseSeverity": "MEDIUM",
                        "confidentialityImpact": "LOW",
                        "exploitCodeMaturity": "UNPROVEN",
                        "integrityImpact": "NONE",
                        "privilegesRequired": "LOW",
                        "remediationLevel": "OFFICIAL_FIX",
                        "reportConfidence": "CONFIRMED",
                        "scope": "UNCHANGED",
                        "temporalScore": 3.8,
                        "temporalSeverity": "LOW",
                        "userInteraction": "NONE",
                        "vectorString": "CVSS:3.0/A:N/UI:N/I:N/PR:L/AV:N/S:U/C:L/AC:L/RL:O/E:U/RC:C",
                        "version": "3.0"
                    }
                }
            ],
            "problemTypes": [
                {
                    "descriptions": [
                        {
                            "description": "Obtain Information",
                            "lang": "en",
                            "type": "text"
                        }
                    ]
                }
            ],
            "providerMetadata": {
                "dateUpdated": "2020-09-22T13:55:41",
                "orgId": "9a959283-ebb5-44b6-b705-dcc2bbced522",
                "shortName": "ibm"
            },
            "references": [
                {
                    "tags": [
                        "x_refsource_CONFIRM"
                    ],
                    "url": "https://www.ibm.com/support/pages/node/6335281"
                },
                {
                    "name": "ibm-idna-cve20204612-info-disc (184924)",
                    "tags": [
                        "vdb-entry",
                        "x_refsource_XF"
                    ],
                    "url": "https://exchange.xforce.ibmcloud.com/vulnerabilities/184924"
                }
            ]
        }
    },
    "cveMetadata": {
        "assignerOrgId": "9a959283-ebb5-44b6-b705-dcc2bbced522",
        "assignerShortName": "ibm",
        "cveId": "CVE-2020-4612",
        "dateReserved": "2019-12-30T00:00:00",
        "dateUpdated": "2020-09-22T13:55:41",
        "state": "PUBLISHED",
        "datePublished": "2020-09-22T13:55:41.717657Z"
    },
    "dataType": "CVE_RECORD",
    "dataVersion": "5.0"
}
